course/assets/js/close_toggle_button.js

```diff
@@ -0,0 +1,17 @@
+// @ts-nocheck
```

Comment:
Why does this code need to be in its own JS file? Couldn't we just put this function in `course_info_toggle.js` and then in `course.ts` do `import { closeToggleButton } from "./js/course_info_toggle"`?

Reply:
Removed unnecessary file